src/Symfony/Bridge/Doctrine/Types/UlidType.php

```diff
@@ -15,9 +15,11 @@
 
 final class UlidType extends AbstractUidType
 {
```

Comment:
not 100% necessary but doctrine/dbal has a Types class where all the type constants are collected, maybe something similar could be done here?

Reply:
Would be:
```php
namespace Symfony\Bridge\Doctrine\Types;
final class Types
{
    public const ULID = 'ulid';
    public const UUID = 'uuid';
    private function __construct() {}
}
```
Any objections? I don't really have any pros or cons for this over a constant directly in the type class, so I'd like to leave this decision to people with more experience.

Reply:
to be honest I don't have a real preference either. Maybe someone from the Symfony team can decide? Both have advantages. 
I only mentioned it because doctrine itself has a types class for the constants.